### ISSUE TITLE
[A11y] General Mobile - Focus (hamburger menu)

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1811]
### Operating System

Windows

### Browser

Edge

### Affected page

Homepage

### Admin user?

Yes

### Issue Description

The "hamburger menu" in mobile doesn't have a visible focus indicator

### Suggested Fix

Undo whatever is hiding the focus indicator - probably something in CSS explicitly stating no border or a white border or something.
